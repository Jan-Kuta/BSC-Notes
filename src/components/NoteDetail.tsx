import React from 'react';
import {NavLink} from 'react-router-dom';

export const NoteDetail = () => (
    <>
        <NavLink to="/1/edit">Edit</NavLink>&nbsp;
        <NavLink to="/">Home</NavLink>
        <h1>NOTE DETAIL</h1>
    </>
);
