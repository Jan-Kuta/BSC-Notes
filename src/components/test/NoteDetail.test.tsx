import React from 'react';
import renderer from 'react-test-renderer';
import {MemoryRouter} from 'react-router';

import {NoteDetail} from '../NoteDetail';

it('NoteDetail renders correctly', () => {
    const tree = renderer
        .create(
            <MemoryRouter>
                <NoteDetail />
            </MemoryRouter>
        )
            .toJSON();
    expect(tree).toMatchSnapshot();
});
