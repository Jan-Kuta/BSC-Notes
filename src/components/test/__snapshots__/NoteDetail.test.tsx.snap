// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoteDetail renders correctly 1`] = `
Array [
  <a
    aria-current={null}
    href="/1/edit"
    onClick={[Function]}
  >
    Edit
  </a>,
  "Â ",
  <a
    aria-current="page"
    className="active"
    href="/"
    onClick={[Function]}
    style={Object {}}
  >
    Home
  </a>,
  <h1>
    NOTE DETAIL
  </h1>,
]
`;
